Makefile

```diff
@@ -74,6 +74,27 @@ GOVERALLS = $(GOBIN)/goveralls
 $(GOBIN)/goveralls: | $(BASE) ; $(info  building goveralls...)
 	$Q env GO111MODULE=off go get github.com/mattn/goveralls
 
+MOCKERY = $(shell pwd)/bin/mockery
+mockery: ## Download mockery if necessary.
+	$(call go-get-tool,$(MOCKERY),github.com/vektra/mockery/v2@v2.8.0)
+
+.PHONY: generate-mocks
+generate-mocks: mockery
+	$(MOCKERY) --srcpkg ./pkg/manager/ --output ./pkg/manager/mocks --all
```

Comment:
This is not the only folder which contains mocks. I think we should add full list here or remove generate-mocks target entirely.